<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogstaff</class>
 <widget class="QDialog" name="Dialogstaff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hellow, staff!</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>231</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>Animals' imformation</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>114</width>
      <height>423</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_tiger">
       <property name="text">
        <string>Tiger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_rabit">
       <property name="text">
        <string>Rabit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_mouse">
       <property name="text">
        <string>Mouse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_dog">
       <property name="text">
        <string>Dog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cat">
       <property name="text">
        <string>Cat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_leopard">
       <property name="text">
        <string>Leopard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_eagle">
       <property name="text">
        <string>Eagle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_lion">
       <property name="text">
        <string>Lion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_crocodile">
       <property name="text">
        <string>Crocodile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_swallow">
       <property name="text">
        <string>Swallow</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_hello">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hello, staff !</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>611</width>
     <height>271</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Imformation of animals

Thanks for using this program
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_getboss">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>520</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>611</width>
     <height>107</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Get imformtions of your boss</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
